docs/operator/operations.md

```diff
@@ -0,0 +1,100 @@
+# Lifecycle Manager - Operations
+
+## Metrics
+
+Lifecycle Manager metrics are exposed over port `8080` through the metrics endpoint `/metrics`.
+
+The following metrics are exposed:
+
+| Metric Name                              | Metric Type    | Description                                                                                  |
+|------------------------------------------|----------------|----------------------------------------------------------------------------------------------|
+| `lifecycle_mgr_requeue_reason_total`     | Counter Vector | Indicates the requeue reason of a reconciler.                                                |
```

Comment:
Please elaborate on the terminology here. "of a reconciler" is an exact description for you, who are working on this stack, however for other people (who do know know the kyma/klm architecture deeply by heart) this is a statement, which is understood as words, but not as an architecture of a stack. I guess this means here, that a module reconciler on an SKR.